lib/faraday/adapter/net_http_persistent.rb

```diff
@@ -185,7 +185,13 @@ def request_with_wrapped_block(http, env)
         min_version: :min_version,
         max_version: :max_version,
         verify_hostname: :verify_hostname
-      }.freeze
+      }
+
+      if Gem::Version.new(Faraday::VERSION) >= Gem::Version.new('2.11.0')
```

Comment:
```suggestion
      if Gem::Version.new(Faraday::VERSION) >= Gem::Version.new("2.11.0")
```

I forgot we use `standard.rb` for code style in this repo, it's complaining about the use of single quotes here. Could you change it to double quotes?